src/commands/react-native/xcode.ts

```diff
@@ -187,7 +187,7 @@ export class XCodeCommand extends Command {
   }
 
   private bundleReactNativeCodeAndImages = async () => {
-    const bundleJSChildProcess = spawn(this.scriptPath, undefined, {
+    const bundleJSChildProcess = spawn(this.scriptPath, {
```

Comment:
The only `spawn` overloads that matches `undefined` args prevents the usage of `inherit` in the `stdio` option (`SpawnOptionsWithoutStdio `):
```ts
function spawn(command: string, args?: ReadonlyArray<string>, options?: SpawnOptionsWithoutStdio): ChildProcessWithoutNullStreams;
```

I think it may be a shortcoming of `@types/node`, but otherwise `options` can be used as the second argument of spawn if there are no args (or we can use `[]` as args).

Reply:
I'd be a bit more comfortable with the `[]` as args for the sake of consistency within this file, but that's a nitpick. If you want to merge as it is, fine by me 👍 

```suggestion
    const bundleJSChildProcess = spawn(this.scriptPath, [], {
```